target/linux/ramips/dts/mt7620n_hootoo_tm05.dts

```diff
@@ -0,0 +1,158 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "mt7620n.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "hootoo,tm05", "ralink,mt7620n-soc";
+	model = "HooToo TM05";
```

Comment:
Vendor calls it as "HooToo Filehub HT-TM05" on their [Travel Router page](https://www.hootoo.com/c/travlrouter)
![image](https://user-images.githubusercontent.com/568751/88405875-27a15e80-cdd0-11ea-9494-912791f30a06.png)

On the [product page](https://www.hootoo.com/p/filehub) there is text: "Model: HT-TM05"

In their User Manual ("User Guide 1.0 - HT-TM05 TripMate.pdf"), on the opening page is only "HooToo" and "HT-TM05", later in "Package Contents" it calls as "HT-TM05 TripMate"


@arrmo , what about introducing this device as "HooToo HT-MT05"?
```suggestion
	compatible = "hootoo,ht-tm05", "ralink,mt7620n-soc";
	model = "HooToo HT-TM05";
```

It would be nice to include the term "FileHub" too but this device was "TripMate" earlier.

Reply:
NP at all! Perhaps the model should be "HooToo FileHub - TripMate Titan"? They seem to use that Titan name in several places.

Just let me know - no issue updating this, agree with you. BTW, several other files need to be updated then (NP, I can take care of it).

Reply:
I'd go with the suggested change; don't forget to update everything to hootoo,ht-tm05 and hootoo_ht-tm05 ...

Reply:
Agreed - that's what I mean by the other files. NP. And the name, include TripMate Titan, like they seem to call out?

Thanks!

Reply:
That "Tripmate Titan" is good enough on the Wiki page, IMHO

Reply:
NP, works for me. Let me try to find all the places this has an impact, update the push here.

And the fun of git, need to (then) figure out how to get this incorporated in to the ravpower branch ... LOL.

Reply:
Updated, re-built => sysupgrade (had to force, the change in name was flagged ... perfect!). Rebooted, all good 😄.

New push coming, just yell if you see anything else!

Now, back to the RAVPower RP-WD03, which follows on from this one (and I need to figure out how to get these changes pulled in to that one ... I really am such a git-idiot 🤣).

Thanks!